### PR TITLE
[DOCS] Add missing `indices` parms returned by `_nodes/stats`

### PR DESCRIPTION
**Pull request by jrodewig**
### Changes
* Adds several human-readable `indices` parameters returned by the
`_nodes/stats` API.
* Syncs several parameter definitions with those in the [cluster stats API docs][0].

[0]: https://www.elastic.co/guide/en/elasticsearch/reference/master/cluster-stats.html

### CODE REVIEW

File: docs/reference/cluster/nodes-stats.asciidoc [hunk 1]
```diff
@@ -446,77 +535,147 @@ in the query cache.
...
+Total amount of memory used by all version maps across all shards assigned to
+the node.
 
 `indices.segments.version_map_memory_in_bytes`::
 (integer)
-Memory, in bytes, used by the version map.
+Total amount of memory, in bytes, used by all version maps across all shards
+assigned to the node.
+
+`indices.segments.fixed_bit_set_memory`::
```
Comment (jbaiera) [inline docs/reference/cluster/nodes-stats.asciidoc]: This should just be `fixed_bit_set`

**Review by jbaiera — APPROVED** [review]
LGTM, one small correction

File: docs/reference/cluster/nodes-stats.asciidoc (hunk 1 above)
Comment (jrodewig) [reply docs/reference/cluster/nodes-stats.asciidoc]: Good catch. Thanks @jbaiera.

**Comment by jrodewig** [comment]
### Backport commits
master 8c50523
7.x 068181b0b68
7.6 c5d7d0b1a31

**Pull request merged**